lib/src/config/route/app_routes.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+import 'package:spoonacular/src/core/util/routes_constants.dart';
+import 'package:spoonacular/src/core/util/strings_constants.dart';
+import 'package:spoonacular/src/presentation/view/home_page.dart';
+import 'package:spoonacular/src/presentation/view/splash_screen.dart';
```

Comment:
relative imports?

Reply:
It marks me as a warning, otherwise I put relative imports?
![image](https://user-images.githubusercontent.com/84338449/199530387-ee024d66-c1b9-4ce2-b93b-18cd1bc82438.png)

Reply:
For the RoutesConstants file as well?